sdk/src/platform/Platform.ts

```diff
@@ -451,6 +455,10 @@ export default class Platform extends EventEmitter {
             const {response} = e;
             const {status} = response;
 
+            if (this._handleRateLimit) {
+                options.handleRateLimit = this._handleRateLimit;
+            }
+
```

Comment:
No need for this since

```js
const {retry, handleRateLimit} = options;
```

Previous options should already have it: you need to place this code in `inflateRequest` method, check if user provided the option and if not set the default one.